tests/config/test_data.config

```diff
@@ -265,6 +266,9 @@ params {
                 test2_haplotc_ann_vcf_gz                       = "${test_data_dir}/genomics/homo_sapiens/illumina/gatk/haplotypecaller_calls/test2_haplotc.ann.vcf.gz"
                 test2_haplotc_ann_vcf_gz_tbi                   = "${test_data_dir}/genomics/homo_sapiens/illumina/gatk/haplotypecaller_calls/test2_haplotc.ann.vcf.gz.tbi"
 
+                test2_haplotc_vcf_gz                           = "${test_data_dir}/genomics/homo_sapiens/illumina/gatk/haplotypecaller_calls/test2_haplotc.vcf.gz"
```

Comment:
What's the difference between the new files and the ones above it?

Reply:
Nothing actually, just the name is different. I just needed to similar VCF files to perform the tests.

Reply:
It's not best practice to clutter the test data repo with unneeded data.
Has this been added to `test_data`? If so, please remove it again.

You can compare the vcf from test1 and test2 for the sake of testing the tool. The ouput doesn't matter.

Reply:
This file was already in the test-datasets repo. But from the output of the benchmark I concluded they are the same file, probably one with annotations and one without?

Reply:
no problemo then!